executor/adapter.go

```diff
@@ -709,7 +709,7 @@ func FormatSQL(sql string, sessVars *variable.SessionVars) string {
 }
 
 // LogSlowQuery is used to print the slow query in the log files.
-func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool) {
+func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, isSelect bool) {
```

Comment:
How about logging other query's plan? For example, the explain result of `update` and `delete` is supported, and the results can be useful.

Reply:
Done.